opta/core/kubernetes.py

```diff
@@ -236,14 +236,14 @@ def create_namespace(layer_name: str) -> None:
         )
 
 
-def create_manual_secrets(layer_name: str) -> None:
+def create_manual_secrets_if_not_exists(layer_name: str) -> None:
     load_kube_config()
     v1 = CoreV1Api()
     manual_secrets: V1SecretList = v1.list_namespaced_secret(
```

Comment:
Wouldn't this error if namespace doesn't exist?

Reply:
which is why we call create_namespace_if_not_exists in secrets.py prior to the update_manual_secrets call